src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -85,9 +110,8 @@ public static BufferedImage makeTextMenu(Color bg, Color frame, Color focus, Arr
       int hBuffer, int vBuffer)
   {
     // Find the dimensions of the menu we are drawing.
-    int drawScale = SpriteOptions.getDrawScale();
-    int menuTextWidth = SpriteLibrary.getLettersSmallCaps().getFrame(0).getWidth() * drawScale;
-    int menuTextHeight = SpriteLibrary.getLettersSmallCaps().getFrame(0).getHeight() * drawScale;
+    int menuTextWidth = SpriteLibrary.getLettersSmallCaps().getFrame(0).getWidth();
+    int menuTextHeight = SpriteLibrary.getLettersSmallCaps().getFrame(0).getHeight();
     int menuWidth = (( items.isEmpty() ) ? 0 : getMenuTextWidthPx(items, menuTextWidth)) + hBuffer * 2;
     int menuHeight = (( items.isEmpty() ) ? 0 : getMenuTextHeightPx(items, menuTextHeight)) + vBuffer * 2;
```

Comment:
`getMenuTextHeightPx()` uses the draw scale. Makes the menus unsightly.

Reply:
This is still an issue
![image](https://user-images.githubusercontent.com/8674241/58780137-a66c8400-859d-11e9-9bc8-7545d16a88cf.png)

Reply:
Yes. Still working, man. Patience.